Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1825,6 +1828,52 @@ namespace PlayerCommands
 		}
 	}
 
+	void GetNecessitiesStatus(uint client, const wstring &args) {
+		// Check that this player is in a player controlled base
+		PlayerBase *base = GetPlayerBaseForClient(client);
+		if (!base)
+		{
+			PrintUserCmdText(client, L"ERR Not in player base");
+			return;
+		}
+
+		if (!clients[client].admin && !clients[client].viewshop)
+		{
+			PrintUserCmdText(client, L"ERROR: Access denied");
+			return;
+		}
+
+		if (base->HasMarketItem(set_base_crew_type) < base->base_level * 200) {
+			PrintUserCmdText(client, L"WARNING, CREW COUNT TOO LOW");
+		}
+		PrintUserCmdText(client, L"Crew: %u onboard", base->HasMarketItem(set_base_crew_type));
+
+		PrintUserCmdText(client, L"Crew supplies:");
+		for (map<uint, uint>::iterator i = set_base_crew_consumption_items.begin(); i != set_base_crew_consumption_items.end(); ++i) {
+			const GoodInfo *gi = GoodList::find_by_id(i->first);
+			if (gi)
+			{
+				PrintUserCmdText(client, L"|    %s: %u", HkGetWStringFromIDS(gi->iIDSName).c_str(), base->HasMarketItem(i->first));

```

Comment:
Why do I feel like there was some horrible nasty gotcha with HkGetWStringFromIDS or two at some point... Never mind, should be fine.

Reply:
This part I shamelessly copied from the ShowShopStatus which renders the /shop window. Fixing something that isn't broken is something I try to avoid :)

Reply:
Oh I think there's something broken about it, but we'll keep the function call.